src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -14,9 +14,7 @@ def __init__(self):
 class PimFileTransferService:
     """Instantiates a file uploader and downloader to have a seamless file reading /
     writing in the cloud particularly in Ansys lab . Here we are exposing upload and
-    download methods on session objects. These would be no- ops if PyPIM is not
-    configured or not authorized with the appropriate service. This will be used for
-    internal purpose only.
+    download methods on session objects. This will be used for internal purpose only.
```

Comment:
_Provides a file transfer service based on PyPIM and simple_upload_server_ is sufficient.